### ISSUE TITLE
Navigating back in history throws error in Safari

### ISSUE THREAD

**Issue by palpatine1991** [DevExpress/testcafe#4395]
### What is your Test Scenario?
Using Client Function `goBack = ClientFunction(() => window.history.back());` breaks the DOM queries in Safari.
 
### What is the Current behavior?
It throws error:

```
 1) TypeError: Cannot read property '0' of null

      Browser: Safari 13.0.2 / Mac OS X 10.15.0

         491 |    const element = Selector('#developer-name');
         492 |    await t.navigateTo(`http://google.com`)
         493 |
         494 |    await ToolkitHelper.goBack();
         495 |
       > 496 |    await t.expect(element.exists).ok();
         497 |});
         498 |

         at Object.<anonymous> (C:\Users\vsipek\WebstormProjects\Common_IDE\e2e\form-system\dropdown\dropdown.desktop.e2e-spec.ts:496:28)
         at fulfilled (C:\Users\vsipek\WebstormProjects\Common_IDE\e2e\form-system\dropdown\dropdown.desktop.e2e-spec.ts:4:58)

```
 
### What is the Expected behavior?
It should work eve after navigating in history.
 
### What is your web application and your TestCafe test code?

 
Your website URL (or attach your complete example):
<details>
<summary>Your complete test report:</summary>

```

 1) TypeError: Cannot read property '0' of null

      Browser: Safari 13.0.2 / Mac OS X 10.15.0

         491 |    const element = Selector('#developer-name');
         492 |    await t.navigateTo(`http://google.com`)
         493 |
         494 |    await ToolkitHelper.goBack();
         495 |
       > 496 |    await t.expect(element.exists).ok();
         497 |});
         498 |

         at Object.<anonymous> (C:\Users\vsipek\WebstormProjects\Common_IDE\e2e\form-system\dropdown\dropdown.desktop.e2e-spec.ts:496:28)
         at fulfilled (C:\Users\vsipek\WebstormProjects\Common_IDE\e2e\form-system\dropdown\dropdown.desktop.e2e-spec.ts:4:58)

```
</details>

### Steps to Reproduce:
```js
fixture`History testing`
    .page(`https://devexpress.github.io/testcafe/example/`)

test.('Should navigate forth and back and check element', async t => {
    const element = Selector('#developer-name');
    const goBack = ClientFunction(() => window.history.back());
    await t.navigateTo(`http://google.com`)
   
    await goBack();

    await t.expect(element.exists).ok();
});
```
 
### Your Environment details:
 
* testcafe version:                   1.6.0
* node.js version:                    10.16.0
* command-line arguments:  testcafe remote --qr-code
* browser name and version: Safari 13.0.2 / 
* platform and version:          Mac OS X 10.15.0

Works as expected on latest Chrome on both Windows and Mac

**Comment by alexey-lin** [comment]
Hi @palpatine1991,

Thank you for contacting us. I've managed to reproduce the issue. We'll look into it and update this thread once we have any results.

**Comment by moh0040** [comment]
This issue can no be reproduce with testcafe version `1.8.1` on `Safari 13.0.2 / Mac OS X 10.15.0`

**Comment by palpatine1991** [comment]
I can confirm, that this issue is resolved in Testcafe `1.8.1`!
